GUI/SearchTaskWindow.xaml.cs

```diff
@@ -600,7 +599,7 @@ private void NonSpecificUpdate(object sender, TextChangedEventArgs e)
                 try
                 {
                     System.Windows.Controls.TextBox textBox = (TextBox)sender;
-                    if (textBox.Name.Equals("txtMaxPeptideLength")) //if maxPeptideLength was modified
+                    if (textBox.Name.Equals("MaxPeptideLengthTextBox")) //if maxPeptideLength was modified
```

Comment:
Change this back. You're trying to compare a string and a textbox, which won't work.